content/en/Integrations/webhooks.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Get Pentest Updates with Webhooks"
+linkTitle: "Webhooks"
+weight: 10
+description: >
+  Set up webhook notifications for your organization.
+---
+
+{{% pageinfo %}}
+Configure webhooks to subscribe to real-time updates for your pentests.
+{{% /pageinfo %}}
+
+With our API-based webhooks, you can set up an integration between your app and the Cobalt platform to get notifications for pentest events. We’ll send you updates for each event to your URL through an HTTP POST request.
+
+When you work with an API, you can become aware of new data in the following ways:
+
+- Repeatedly send requests to the same [API endpoint](/getting-started/glossary/#api-endpoint) to retrieve new information. This process is known as API polling.
+- Configure a webhook that automatically sends updates for new data to the specified URL.
+
+{{< mermaid align="left" theme="default" >}}
+sequenceDiagram
+  title Webhook for pentest events
+   Your App->>Cobalt Platform: Notify me of new pentest events
+   Cobalt Platform-->>Your App: Pentest created
+   Cobalt Platform-->>Your App: Finding created
+   Cobalt Platform-->>Your App: Asset updated
+   Cobalt Platform-->>Your App: Finding state changed
+{{< /mermaid >}}
+<br>
+
+You can configure webhooks through the API and in the [Cobalt UI](#configure-webhooks-in-the-ui).
+<!-- Provide a link to API docs or API use case. -->
+
+## Before You Start
+
+Before you start creating webhooks, complete the configuration in your app.
+
+1. Generate an **authentication token**. We need this token to authorize HTTP POST requests to your URL and connect to your app.
+1. Make sure that the **URL** where you want to receive notifications is valid and your services work properly.
```

Comment:
I wonder if we can/should say something like "If possible, test your connections"

Functionally, I'm concerned about firewalls which filter certain kinds of data (or stops data that's received frequently)